src/app/shell/personal-cabinet/parent/create-child/create-child.component.ts

```diff
@@ -65,7 +67,7 @@ export class CreateChildComponent extends CreateFormComponent implements OnInit,
     
     if (!this.editMode) { 
       this.ChildrenFormArray.push(this.newForm());     
-    }
+    } else this.AgreementFormControl.setValue(true);
```

Comment:
this.editMode ?
  this.ChildrenFormArray.push(this.newForm()) : 
  this.AgreementFormControl.setValue(true);